server/pages/index.js

```diff
@@ -391,12 +394,16 @@ module.exports = function() {
 				: 'all';
 
 			if ( config.isEnabled( 'server-side-rendering' ) ) {
+				const store = createReduxStore();
+				store.dispatch( setSection( 'design', { hasSidebar: false } ) );
+				context.initialReduxState = pick( store.getState(), 'ui' );
+
```

Comment:
I think we want to put these lines into the `try` block, so the worst thing that happens if store creation or `dispatch` fail is an empty SSR'ed layout, with the client having to take over rendering duties, instead of the server 500ing.